config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -7,6 +7,7 @@ periodics:
     fork-per-release-generic-suffix: "true"
     testgrid-dashboards: sig-release-master-blocking, sig-node-kubelet
     testgrid-tab-name: node-kubelet-master
+    testgrid-alert-email: kubernetes-sig-node@googlegroups.com
```

Comment:
suggest putting a + in the email so that it's easier to label:

e.g.

kubernetes-sig-node+prow@googlegroups.com

Reply:
will use kubernetes-sig-node+testgrid@

FWIW the subject / from-email is pretty consistent for labeling purposes, see https://groups.google.com/forum/#!forum/kubernetes-sig-network-test-failures for example